### DISCUSSION TITLE
Weird path behavior

### DISCUSSION THREAD

**Discussion by bkeph** [pmndrs/gltfjsx#141] [General]
I have used npx gltfjsx passing a glt file, then I have moved the glt file and adapted the path passed to `useGLTF()` and `useGLTF.preload()`, but it seems that it cannot find it in the new folder, showing a GET error:

`GET http://localhost:3200/path_to_file/file.glb 404 (Not Found)`

However, if I try to load the file from its initial path (where it does not reside anymore), it works.

**Comment by drcmda** [comment]
im afraid this isnt related to gltfjsx, it does nothing to the file, it doesnt load it, it just creates a blob of text. you need to put models into the public folder and access them with an absolute path starting with "/", like all other static files.
